integrationtests/test_all_examples.py

```diff
@@ -346,11 +346,26 @@ def test_run(self):
                     )
                 )
 
-
+@requires_toolkit([ToolkitName.qt, ToolkitName.wx])
 class TestInteractExample(unittest.TestCase):
     """ Test examples with more interactions."""
 
-    @requires_toolkit([ToolkitName.qt, ToolkitName.wx])
+    def test_button_editor_simple_demo(self):
+        # Test ButtonEditor_simple_demo.py in examples/demo/Standard_Edtiors
+        filepath = os.path.join(
+            DEMO, "Standard_Editors", "ButtonEditor_simple_demo.py"
+        )
+        demo = load_demo(filepath, "demo")
+
+        tester = UITester()
+        with tester.create_ui(demo) as ui:
+            button = tester.find_by_name(ui, "my_button_trait")
+            for _ in range(5):
+                button.perform(command.MouseClick())
```

Comment:
I'd suggest adding `self.assertEqual(demo.click_counter, index + 1)` following the mouse click.
This demonstrates one of the two common testing patterns: Modify GUI state resulting in trait changes

Likewise, adding `demo.click_counter = 10` (deliberately not 5) and then inspect the displayed value is updated to "10" demonstrates the second of the two common testing patterns: Modify trait updates the GUI state.